ivy_tests/test_ivy/helpers/available_frameworks.py

```diff
@@ -8,7 +9,10 @@ def _available_frameworks():
     for backend in ["numpy", "jax", "tensorflow", "torch", "paddle"]:
         if find_spec(backend) is not None:
             ret.append(backend)
-    return ret
+        elif os.path.exists(f"/opt/fw/{backend}"):
```

Comment:
`/opt/fw/` should be stored in a global variable as it seems being used widely.

Reply:
Yeah we sure can do that.